lib/migrate.js

```diff
@@ -42,6 +47,30 @@ function migrate (set, direction, migrationName, fn) {
         next(migrations.shift())
       })
     })
+
+    if (migrationPromise && migrationPromise.then) {
```

Comment:
Prefer this to be `typeof Promise !== 'undefined' && migrationPromise instanceof Promise`